megablocks/layers/dmoe.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, args : Arguments):
         self.hidden_size = args.hidden_size
         self.ffn_hidden_size = mpu.features_per_rank(args)
         self.blocking = 128
+        self.actq_num_bits = args.quantize_activations_num_bits
```

Comment:
nit: this is only reused once, so I'd just pass it from self.args instead of creating another member.